WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -229,6 +238,8 @@ class PostListMainViewModel @Inject constructor(
                 authorFilterSelection = authorFilterSelection,
                 authorFilterItems = getAuthorFilterItems(authorFilterSelection, accountStore.account?.avatarUrl)
         )
+        _previewState.value = _previewState.value ?: PostListRemotePreviewState.NONE
```

Comment:
😊…what was that, some sort of mind reading? 😅...you actually got exactly the path I went through. I needed to reassign the live data for the reason you highlighted. Initially I was tempted to align the start method to the approach with flag I noticed in other VMs, but preferred to stay with just that simple modification of the code I was introducing in case I was missing some rationale behind that 😊.

Reply:
I'm really glad to hear you were thinking about it and that you got to the same conclusion. The decision to go with the simple modification seems reasonable to me in this case ;)!